go/enclave/enclave.go

```diff
@@ -1050,7 +1050,6 @@ func extractGetLogsParams(paramBytes []byte) (*filters.FilterCriteria, *gethcomm
 }
 
 // Removes transactions from the mempool that are considered immune to re-orgs (i.e. over X rollups deep).
-// TODO - #718 - Only pass in header number, not full rollup.
```

Comment:
This wasn't workable in practice, since we need to walk the actual chain back in case of a fork (i.e. we can't just grab the header with number x from the DB).